seedDB/README.md

```diff
@@ -4,7 +4,16 @@ These files are MySQL database dump files for seeding a new instance of the cBio
 
 The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database schema 2.1.0. This means that the version numbers won't be identical. cBioPortal 1.9.0 with database schema 2.4.0 removed PDB annotations from the database.
 
-## Latest seed database
+## Release Notes
+### Latest seed database
+#### Gene Table 2.0
```

Comment:
I would follow format of previous seed database entries (see image below).  In the section of "Contents of seed database" I would put the information about genes (in a single bullet with sub-bullets).

![image](https://user-images.githubusercontent.com/366003/112491731-f6aa6c80-8d56-11eb-97e4-04c75ed76a88.png)

Reply:
I agree with @n1zea144. Maybe the schema can also be updated to 2.12.8 (current version). All new installations are going to work with the latest cBioPortal version, so it does not make sense to keep an old schema. I think that is the reason why we decided to do the seed database updates based on the schema version.

Reply:
@n1zea144 @oplantalech make sense. Just followed the previous format and create a whole new section 2.12.8 with gene table update as a bullet point. (The links to  needs to be updated tho, marked that -- needs further revise once the new mySQL dumps are made)